components/utilities/chartLayout.js

```diff
@@ -149,9 +149,9 @@
             chartElements.chartArea = chart;
 
             // Defs - for clipping path
-            var defs = chart.selectAll('defs').data(dummyData);
-            defs.enter().append('defs');
-            defs.exit().remove();
+            chartElements.defs = chart.selectAll('defs').data(dummyData);
+            chartElements.defs.enter().append('defs');
+            chartElements.defs.exit().remove();
```

Comment:
Stylistically, I am not keen on the repetition of `chartElements.defs` in the above code. Isn't the following equivalent?

       chartElements.defs = chart.selectAll('defs').data(dummyData)
                                                  .enter().append('defs');
                                                  .exit().remove();